composer.json

```diff
@@ -30,7 +30,7 @@
     "require-dev" : {
         "kriswallsmith/buzz" : "~0.8",
         "guzzle/http"        : "~3",
-        "guzzlehttp/guzzle"  : "~5.2"
+        "guzzlehttp/guzzle": "^6.1"
```

Comment:
Something like this will allow the range of versions, Composer doesn't allow more than one version of the same package to be installed at once:
`"guzzlehttp/guzzle": ">=5.2 <6.2.0"`

See [this Travis doco](https://docs.travis-ci.com/user/languages/php#Testing-Against-Multiple-Versions-of-Dependencies-(e.g.-Symfony)) to configure testing against multiple dependency versions.